docker/README.md

```diff
@@ -131,6 +138,7 @@ $ docker push polarlightning/clightning:<version>
 ```sh
 $ cd eclair
 $ docker build --build-arg ECLAIR_VERSION=<version> -t polarlightning/eclair:<version> .
+$ docker buildx build  --platform linux/amd64,linux/arm64 --build-arg ECLAIR_VERSION=0.6.2 -t polarlightning/eclair:0.6.2  --push  .
```

Comment:
Remove the `docker build` line above this one and change `0.6.2` to `<version>`.

Reply:
I have done this, please review

Reply:
Confirmed 👍